### PR TITLE
Alternative HSR policy

### PR DESCRIPTION
**Pull request by Keno**
This is an alternative to https://github.com/JuliaLang/www.julialang.org/pull/1229. After some extensive discussion on
the committers channel, the following objections were raised to the original draft proposal:

1. Such research is still somewhat rare, and despite the recent high profile incidents, it seems unwise to draft full policies without a concrete need in this community
2. Requiring notification to stewards was not universally regarded as a good idea. Everybody agreed that Stewards cannot give consent to perform HSR on behalf of the participants. While this is not what was contemplated in the original PR, the concern was that any sort of advance review would subsequently be perceived as implied approval and we do not want to put Stewards into this position.

This instead amends the CoC, with the goal of explicitly putting researchers on notice that we as a community consider such research to be research on human subjects and therefore needs to be treated as such, and if you're behaving unethically you may be banned. There was a remaining question about whether such a statement was actually required, considering that we would definitely ban people for misbehavior regardless of whether or not they were a researcher. However, I would like to think that most researchers at least intend to act ethically, so I'm hoping that an explicit notice of this sort would give those who do intend ethically a chance to examine their research and consider whether they may need to follow applicable HSR practices before proceeding in their research.

### CODE REVIEW

**Review by StefanKarpinski — APPROVED** [review]

File: community/standards.md
Comment (DilumAluthge) [inline community/standards.md:23]:
```suggestion
We welcome the participation of researchers of all kinds in the Julia community. However, please remember that open source development is a social process and that the participants on the other side of any interaction are (obviously) humans. If you are doing research *on the community or its processes*, you are likely performing Human Subjects Research (HSR) and are expected to abide by the highest standards of ethics and professional practice. Meeting this expectation is your responsibility. In particular, please be aware that Institutional Review Boards (IRBs) or other review committees may not have sufficient context or experience to understand the social processes of an open source community. We consider engaging in unethical research (human subject or otherwise) to be a cause for bans or other sanctions from the community.
```

**Review by JeffBezanson — APPROVED** [review]

**Review by KristofferC — APPROVED** [review]

**Comment by Keno** [comment]
This wording appears to be agreeable to the committers who engaged in the previous discussion. I'm leaving this open for a few days to encourage further community feedback, but absent any significant objection, I will plan to merge this in a few days. ccing participants on the discussion in #1229, to make sure they have a chance to chime in: @logankilpatrick @oxinabox @jiahao @simonbyrne

**Comment by jdossgollin** [comment]
This is tangential to the specific question of security research, but this was my first time reading through the community standards and I noticed that the language on inclusion in the community is rather vague. Is there a reason that building a diverse and inclusive community is not explicitly listed as a value on this page?

**Comment by Keno** [comment]
> This is tangential to the specific question of security research, but this was my first time reading through the community standards and I noticed that the language on inclusion in the community is rather vague. Is there a reason that building a diverse and inclusive community is not explicitly listed as a value on this page?

Revising that section is #1235 - there's been some discussion.

**Comment by jdossgollin** [comment]
> Revising that section is #1235 - there's been some discussion.

Thanks, my comment definitely belongs there.

**Comment by jpfairbanks** [comment]
Maybe it makes sense to include along with the ban, a notification of your IRB. Something to make clear that you might think you don't need IRB approval, but if members of the community think you do, they will report you to the relevant authority. That authority starts at the IRB and goes up to the funding agencies. Institutions are incentivized to enforce IRB procedures because of the collateral consequences on their biological and behavioral science programs that receive government funding. If the research is problematic enough to require a ban, it probably requires that we take action to report it. Being reported for violating the terms of your employment agreement carries a much bigger penalty than a Julia community ban. So I think it makes sense to put them on notice of that too.

**Review by vchuravy — APPROVED** [review]

**Review by ViralBShah — APPROVED** [review]

**Pull request merged**
